composer.json

```diff
@@ -43,5 +47,16 @@
     "config": {
         "sort-packages": true
     },
-    "prefer-stable": true
+    "prefer-stable": true,
+    "minimum-stability": "dev",
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/magento/quality-patches.git"
+        },
+        {
+            "type": "vcs",
+            "url": "https://github.com/magento-mpi/magento-cloud-patches.git"
```

Comment:
Should it be main Magento org?

Reply:
It's a temporary solution, to make it work. composer.json will be updated later when we will have packages on repo.com, and logic related to package replacement will be moved to Travis config. Please, skip a review of composer.json now

Reply:
composer.json has been updated, repositories were removed